ngraph/test/backend/mish.in.cpp

```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <algorithm>
+#include <cinttypes>
+#include <cmath>
+#include <cstdlib>
+#include <random>
+#include <string>
+
+#include "gtest/gtest.h"
+#include "runtime/backend.hpp"
+#include "ngraph/runtime/tensor.hpp"
+#include "ngraph/ngraph.hpp"
+#include "util/all_close.hpp"
+#include "util/all_close_f.hpp"
+#include "util/ndarray.hpp"
+#include "util/test_control.hpp"
+#include "util/test_tools.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+static string s_manifest = "${MANIFEST}";
+
+template <element::Type_t Type, typename T = fundamental_type_for<Type>>
+void mish_test(const PartialShape& dynamic_shape,
```

Comment:
DO NOT MERGE Please!.  There is an issue, without "static" keyword, the IE_CPU/IE_GPU backend test would call mish_test from "INTERPRETER" backend, adding static back we can see IE_CPU test will be failed, I'm trying to fix it right now.

Reply:
according to Ilya IE-backend fp16 test is not working right now, so I put them into disabled test list, and also added "static" linkage keyword back